README.md

```diff
@@ -1,54 +1,40 @@
-## **How to install Cesium Gem**
+![Cesium for O3DE Logo](Documentation/Images/Cesium_for_O3DE_dark_onWhite.jpg)
 
-### **Prerequisite**
+Cesium for O3DE brings the 3D geospatial ecosystem to the Open 3D Engine. Combine a high-accuracy full-scale WGS84 globe, open APIs and open standards for spatial indexing such as 3D Tiles, and cloud-based real-world content from [Cesium ion](https://cesium.com/cesium-ion) with the Open 3D Engine.
```

Comment:
Use `Open 3D Engine` only once in this readme. All other instances should be O3DE.